tests/subsys/settings/functional/file/prj_nrf52840_pca10056.conf

```diff
@@ -0,0 +1,26 @@
+CONFIG_ARM_MPU=n
```

Comment:
Is this done for being able to write to flash?
If yes, have you considered the option that allows flash write?

Reply:
@nvlsianpu can you reply?

Reply:
@ioannisg Do you mean "MPU allow flash write option"?

Reply:
I opened another PR for [select to allow flash write by MPU on nRF devices](https://github.com/zephyrproject-rtos/zephyr/pull/20314).